### PR TITLE
nb: 7.14.4 -> 7.14.6

### PR DESCRIPTION
**Pull request by kachick**
Changes: https://github.com/xwmx/nb/compare/7.14.4...7.14.6

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by kachick** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 356336`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>nb</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>nb</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>nb</li>
  </ul>
</details>

**Comment by kachick** [comment]
@toonn Could you review?

File: pkgs/by-name/nb/nb/package.nix [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
   pname = "nb";
-  version = "7.14.4";
+  version = "7.14.6";
 
   src = fetchFromGitHub {
     owner = "xwmx";
     repo = "nb";
     rev = version;
-    sha256 = "sha256-YqqZZnin+aybAZ2dqaxdOrVZ7dLWwnjh2iL77orqHtE=";
+    sha256 = "sha256-VcYE1luwNtdrOmjTDKkIAX9rmcbFyCBtnib+c9FyJuA=";
```
Comment (kirillrdy) [inline pkgs/by-name/nb/nb/package.nix:11]:
```suggestion
    hash = "sha256-VcYE1luwNtdrOmjTDKkIAX9rmcbFyCBtnib+c9FyJuA=";
```

`hash` is preferred


https://nixos.org/manual/nixpkgs/unstable/#fetchfromgithub

**Comment by kirillrdy** [comment]
@kachick similar comment about commit message

**Comment by kachick** [comment]
Thanks for reviewing!

I have updated with reviewed points except for squashing the commits. Because I don't have confidence that including refactoring changes in the version up commit will be better or not. Please tell me if it is required.

**Comment by kirillrdy** [comment]
@ofborg eval

**Comment by kachick** [comment]
Failed `ofborg-eval-lib-tests — nix-build --arg pkgs import ./. {} ./lib/tests/release.nix` from https://github.com/NixOS/nixpkgs/pull/354738, I'll rebase to avoid it.

**Comment by kirillrdy** [comment]
7.15 is out !

**Comment by kachick** [comment]
Thank you for merging...! :joy:

**Pull request merged**